Basic/APIDiorama/Assets/Common/Scripts/com.unity.samples.api-diorama.common.asmdef

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "com.unity.samples.api-diorama.common",
```

Comment:
I lost track of the conversation regarding the names, but I would suggest renaming this namespace to `Unity.Netcode.Samples.APIDiorama.Common` since you can't use dashes inside namespaces names inside of scripts.

I say this since I was expecting scripts in this directory to be using this name, but you can't use `com.unity.samples.api-diorama.common` as a namespace in code because of the dash.

I would extend this to the other classes inside their namespaces. So for example ProximityChecker would be under the `Unity.Netcode.Samples.APIDiorama.Proximity` namespace and so on.

Reply:
> I would extend this to the other classes inside their namespaces. So for example ProximityChecker would be under the Unity.Netcode.Samples.APIDiorama.Proximity namespace and so on.

Done!